src/model/enhance-interactive-atom-elements.ts

```diff
@@ -1,21 +1,29 @@
-import { sanitiseHTML } from '@root/src/model/clean';
+import { sanitiseHTML, clean } from '@root/src/model/clean';
 
 // Some interactives contain HTML with unclosed tags etc. To
 // preserve (approximate) parity with Frontend we perform some basic
 // cleaning.
-export const enhance = (elements: CAPIElement[]): CAPIElement[] => {
+export const enhance = (
+	elements: CAPIElement[],
+	isAmp: boolean = false,
+): CAPIElement[] => {
 	return elements.map((element) => {
 		if (
 			element._type ===
 			'model.dotcomrendering.pageElements.InteractiveAtomBlockElement'
 		) {
-			element.html = element.html
-				? sanitiseHTML(element.html, {
-						ADD_TAGS: ['iframe', 'script'],
-				  })
-				: element.html;
+			if (isAmp) {
```

Comment:
I should have mentioned sorry - one of our principles with AMP/web is to avoid `isAmp` checks and keep separate code separate. This is to make refactoring things easier (we don't want AMP changes to break web for example), and also for easier testing etc.

Really we should move the 'model' stuff under `web/enhancers` or something as it's entirely web-related atm. I'll raise that as a separate PR.

Reply:
Sure - does this mean it should be in a separate file? Or just a separate helper?